plugins/colorbutton/plugin.js

```diff
@@ -187,7 +187,7 @@
 						panelBlock = block;
 						block.autoSize = true;
 						block.element.addClass( 'cke_colorblock' );
-						block.element.setHtml( renderColors( colorBoxId, clickFn, history ? history.getLength() : 0 ) );
+						block.element.setHtml( CKEDITOR.tools.htmlSafeByReview(renderColors( colorBoxId, clickFn, history ? history.getLength() : 0 ), 'renderColors is a template that uses values all created internally.') );
```

Comment:
Can we try to have `renderColors` return a safe type? `TrustedHTML`? 

Also note for CSP refactorings: we should try to get rid of the inline event handlers in `renderColors`

Reply:
`renderColors` will return TrustedHTML.